lib/install.js

```diff
@@ -272,8 +272,10 @@ Installer.prototype.run = function (cb) {
         [this, this.saveToDependencies])
     }
   }
-  postInstallSteps.push(
-    [this, this.printInstalled])
+  if (npm.config.get('loglevel') !== 'silent') {
```

Comment:
Based on https://github.com/npm/npm/pull/11784#issuecomment-195034724 this should be reverted, and the conditional should take into consideration `--json` and `--parseable` flags somewhere closer to [install.js#L687](https://github.com/npm/npm/blob/2005f4ce11f6cdf142f8a77f4f7ee4996000fb57/lib/install.js#L687).